metadata/2024-05-22-transaction-context-extension/enable_transaction_context_extension.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "AIP-82: Allowlisting for delegation pools",
```

Comment:
@junkil-park let's update the dates then we can land this

Reply:
I updated the date to today.

Reply:
I also corrected the title. The other infos are already correct.